nengo/builder/optimizer.py

```diff
@@ -399,67 +719,37 @@ def _perform_merges_for_view_subset(
             Subset of operators. These need to have the same view base (can be
             None if it is None for all) for their first signal in
             `all_signals`.
-        tc : dict
-            Transitive closure of dependency graph.
-        only_merge_ops_with_view : bool
-            Limit merges to operators with views.
-        op_replacements : dict
-            Dictionary that maps from original operators to the newly merged
-            operators. To be used to update the dependency graph. This dict
-            will be updated by this function.
-        sig_replacements : dict
-            Dictionary that maps from original signals to new signals resulting
-            from the merges. To be used to update the signals referenced by
-            the operators. This dict will be updated by this function.
         """
-        view_indices = []
+
         # Sort to have sequential memory.
-        offsets = np.array([self._first_view_offset(op) for op in subset])
+        offsets = np.array([_opinfo[op].v_offset for op in subset])
         sort_indices = np.argsort(offsets)
         offsets = offsets[sort_indices]
         sorted_subset = [subset[i] for i in sort_indices]
 
         for i, op1 in enumerate(sorted_subset):
-            if op1 in self._merged:
+            if op1 in self.merged:
                 # Cannot merge merged operator again until dependency graph
-                # has been updated (happens at the end of one optimization
-                # pass by calling _update_dg).
+                # has been updated
                 continue
-            if any(op1 in tc[op] or op in tc[op1] for op in self._merged):
+
+            if any(op1 in self.tc[op] or op in self.tc[op1]
+                   for op in self.merged):
                 continue
 
-            view_indices = self._get_view_indices(op1)
+            # Find operators to be merged.
```

Comment:
This comment is misleading to me because the next line does not find operators to merge, but just constructs some supporting data structures.